.github/workflows/sync-issues.yml

```diff
@@ -47,6 +47,11 @@ jobs:
           TWITTER_ACCESS_TOKEN: ${{ secrets.TWITTER_ACCESS_TOKEN }}
           TWITTER_ACCESS_TOKEN_SECRET: ${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
         run: npx tsx index.ts
+      
+      - name: Delete Unauthorized Issues
+        run: bash ./scripts/delete-unauthorized-issues.sh
+        env:
+         GH_TOKEN: ${{ secrets.PAT }}  
```

Comment:
Please don't do this with having to add an extra secret PAT when we already have a bot auth token from the previous step. Use the bot auth and I can merge this in. 

```
      - name: Get GitHub App token
        uses: tibdex/github-app-token@v1.7.0
        id: get_installation_token
        with:
          app_id: ${{ secrets.APP_ID }}
          private_key: ${{ secrets.APP_PRIVATE_KEY }}
```

Reply:
Also can you confirm that this script works? I'm pretty sure the default environment variable name is `GITHUB_TOKEN`?